CONTRIBUTING.md

```diff
@@ -216,9 +216,25 @@ Please update the resulting commit message, if needed. It should read as a
 coherent message. In most cases, this means not just listing the interim
 commits.
 
-If you have problems with squashing or other git workflows, you can enable
-"Allow edits from maintainers" in the right-hand sidebar of the GitHub web
-interface and ask for help in the pull request.
+If your change contain a merge commit, the above workflow may not work. You
+have the option to squash everything into *one* commit, or *keep* the commits
+in a non-interactive rebase.
+
+To squash everything into one commit.
```

Comment:
I think keeping this part is enough. By far and wide, a new contributor will want to squash everything into one commit.

Reply:
Thx, fixed.